pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/replication.tsx

```diff
@@ -72,22 +94,54 @@ export default function (props: GraphDashboardProps) {
       </Axis>
     </LineGraph>,
 
-    <LineGraph title="Replica Quiescence" sources={storeSources}>
+    <LineGraph
+      title="Replica Quiescence"
+      sources={storeSources}
+      tooltip={(
+        <div>
+        Number of replicas and quiesced replicas {tooltipSelection}.
+        <br/>
+        Quiesced replicas have no pending reads or writes and have temporarily disabled consensus protocols.
```

Comment:
from @Amruta-Ranade : We should make this clearer to the user, specifically what action needs to be taken when this number is high or low.